examples/jupyter/search.ipynb

```diff
@@ -95,22 +95,22 @@
     "    moderator_cell.region = +clad_outer_radius & (+min_x & -max_x & +min_y & -max_y)\n",
     "\n",
     "    # Create root Universe\n",
-    "    root_universe = openmc.Universe(name='root universe', universe_id=0)\n",
+    "    root_universe = openmc.Universe(name='root universe')\n",
     "    root_universe.add_cells([fuel_cell, clad_cell, moderator_cell])\n",
     "\n",
     "    # Create Geometry and set root universe\n",
     "    geometry = openmc.Geometry(root_universe)\n",
     "    \n",
+    "    # Create an initial uniform spatial source distribution over fissionable zones\n",
+    "    bounds = [-0.63, -0.63, -10, 0.63, 0.63, 10.]\n",
+    "    uniform_dist = openmc.stats.Box(bounds[:3], bounds[3:], only_fissionable=True)\n",
+    "    \n",
```

Comment:
I would leave this block where it originally was. Generally, you want variable definitions to appear closest to where they are used (in this case, creating a `Source` object)